contract/README.rst

```diff
@@ -0,0 +1,67 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+=================================
+Contracts for recurrent invoicing
+=================================
+
+ * This module forward-port to v9 the contracts management with recurring
+   invoicing functions.
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to Sales -> Contracts and select or create a new contract.
+#. Check *Generate recurring invoices automatically*.
+#. Fill fields and add new lines. You have the possibility to use markers in
+   in description field to show init date and end of invoiced period.
+#. To view discount field set *Discount on lines* in user access rights.
+#. A cron is created with daily interval, but if you are in debug mode can
+   click on *Create invoices* to force this action.
+#. Click *Show invoices* link to show all invoices created by the contract.
+#. Press *Invoices* button to show all invoices related with the contract.
```

Comment:
This shouldn't go in this module, because it's an independent feature. Keep it on the original module.